lib/rubocop/chef/inject.rb

```diff
@@ -0,0 +1,16 @@
+module RuboCop
+  module Chef
+    # Because RuboCop doesn't yet support plugins, we have to monkey patch in a
+    # bit of our configuration.
+    module Inject
+      def self.defaults!
+        path = CONFIG_DEFAULT.to_s
+        hash = ConfigLoader.send(:load_yaml_configuration, path)
+        config = Config.new(hash, path)
+        puts "configuration from #{path}" if ConfigLoader.debug?
+        config = ConfigLoader.merge_with_default(config, path)
+        ConfigLoader.instance_variable_set(:@default_configuration, config)
+      end
+    end
```

Comment:
is this duplicated with whatever else in cookstyle that does config injection already?

Reply:
yeah it's totally duplicate. There's a note about that in the PR. I need to rip all this double injection out.